MandoWebApp/ClientApp/src/app/_services/unit.service.ts

```diff
@@ -0,0 +1,16 @@
+import { HttpClient } from '@angular/common/http';
+import { Inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { AddUnit } from '../models/api-inputs/add-unit';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UnitService {
+
+  constructor(@Inject('BASE_URL') public baseUrl: string, private http: HttpClient) {}
+
+  public addUnit(input: AddUnit): Observable<any> {
+    return this.http.post<any>(this.baseUrl + 'unit/addunit', input);
```

Comment:
This could simply be /unit/add or even just a post to /unit, but that's just me nitpicking 🗡️

Reply:
Done, went with the /[controller]/add solution. Also applied to the category controller as well.